packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -12,20 +47,108 @@ export default {
     name: 'ImageTile',
     components: {},
     props: {
+        href: {
+            type: String,
+            default: 'https://www.google.com'
+        },
+        tileId: {
+            type: String,
+            default: 'Chicken'
+        },
+        isSelected: {
+            type: Boolean,
+            default: false
+        },
+        isLink: {
+            type: Boolean,
+            default: false
+        },
```

Comment:
I would remove this prop and check if there's `href` value to check if it's a link. Don't have a strong opinion on this though.

Reply:
I like the separation of having the `isLink` prop. Agree that I could achieve the functionality by checking the `href` value but for me having `isLink` makes it clear the component can have different behaviours.

Reply:
I tend to agree with @1itvinka – the only issue with having a prop for whether it's a link as well as an `href` prop is that you're allowing more room for error for consumers of the component. For someone new to the component, if I add a link to the `href` prop, I'd expect the component to act like a link (I wouldn't expect it to not do that without also setting the `isLink` prop).

For me, `isLink` should be a computed prop which does a check for whether href is set, unless you ever want there to be a use-case where the `href` can be set to a value and `isLink` also set to false.